CODEOWNERS

```diff
@@ -1,9 +1,7 @@
-/plugin/ @mattermost/toolkit
```

Comment:
Maybe makes sense to use @mattermost/integrations for this? cc @hanzei

```suggestion
/plugin/ @mattermost/integrations
```

Reply:
I'm all for @mattermost/integrations taking ownership here! Feel free to commit the suggestion if your team is onboard with that :)